config/development.json

```diff
@@ -1,13 +1,13 @@
 {
 	"extends": "base",
 	"io_sites": {
-		"max_static_paths": 0
+		"max_static_paths": 1
 	},
 	"dev_dot": {
-		"max_static_paths": 0
+		"max_static_paths": 1
 	},
 	"learn": {
-		"max_static_paths": 0
+		"max_static_paths": 1
```

Comment:
Just updated these values to validate in development that everything was working as expected. There should be no functional difference as the static paths are not used in development. 👀